### DISCUSSION TITLE
2012/01/07/%E8%B1%86%E8%B1%862012%E5%85%83%E6%97%A6%E7%9A%84%E7%85%A7%E7%89%87

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#352] [Announcements]
# 2012/01/07/%E8%B1%86%E8%B1%862012%E5%85%83%E6%97%A6%E7%9A%84%E7%85%A7%E7%89%87

这次回无锡没有带上豆豆的专用相机，只用手机抓拍了几张。

https://blog.qizhen.xyz/2012/01/07/%E8%B1%86%E8%B1%862012%E5%85%83%E6%97%A6%E7%9A%84%E7%85%A7%E7%89%87

**Comment by ruanqizhen** [comment]
之前博客下的留言：

7doudou
2012年01月10日 20:31 编辑
好巧，我也叫豆豆- -。
小豆豆，新年快乐~~~

回复
haiying
2012年01月7日 23:03 编辑
小豆豆越长越精神! 爸爸也很帅嘛

回复
ruanqizhen
2012年01月10日 10:45 编辑
豆豆毕竟从他妈那里继承了不少优秀基因，长得比我漂亮多了:)

回复
